### PR TITLE
Allow Fangorn to be ran from local and refactor code following reorganization of repo

### PR DESCRIPTION
**Pull request by Arthurim**
- refactor Fangorn/ to follow the recent changes in the repo
- allow Fangorn to be ran from local

Resolves Issue #11  and #18 

@yvesleconte Please review structure + typos and logic, no need to go through everything and no need to be able to run the code

### CODE REVIEW

File: src/fangorn/nav.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    """
+    borrowing_terms = get_borrowing_terms_fangorn()
+    cash_flows = generate_cash_flows(borrowing_terms)
+    aum = get_current_AUM_fangorn(cash_flows)
+    real_aum = round(aum[aum["Counterparty"] == "Total"]["Amount (USD)"].values[0], 2)
+    print("Total AUM = $", f'{real_aum:,}')
+
+    previous_pnl = get_pnl_spreadsheet_fangorn()
+    previous_pnl['MtM Date'] = pd.to_datetime(previous_pnl['MtM Date'], format=DATE_FORMAT).dt.date
+    last_friday_of_last_week = datetime.datetime.today().date()  # get_last_friday()
```
Comment (yvesleconte) [inline src/fangorn/nav.py:153]: unnecessary comment

**Review by yvesleconte — COMMENTED** [review]
Typos and logic checked

File: src/aeglos/read_write.py [hunk 2]
```diff
@@ -24,8 +25,8 @@ def write_dataframe_to_aeglos(df, sheet):
     if READ_FROM_LOCAL:
         write_dataframe_to_local_sheet_aeglos(df, sheet)
     else:
-        gc = gspread.service_account(filename='C:/dev/amiable-octane-340913-90b8fb61522d.json')
-        sh = gc.open("")
+        gc = gspread.service_account(filename=PATH_TO_PASSWORDS+'amiable-octane-340913-90b8fb61522d.json')
```
[the remark this answers is not in the document]
Comment (Arthurim) [reply src/aeglos/read_write.py:28]: We can have a constant that contains the name of the json file yes but I do not want to get the password and keys out of the json, I think it's better for security to have it fully in a json.

File: src/fangorn/nav.py (hunk 1 above)
Comment (Arthurim) [reply src/fangorn/nav.py]: ✅

File: src/fangorn/nav.py [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+    returns: pd.DataFrame
+    columns: ['Counterparty', "Amount (EUR)", "Amount (USD)"]
+    """
+    return get_aum_at_date(cash_flows, pd.Timestamp(datetime.datetime.now()))
+
+
+def reconciliate_real_aum_and_implied_aum_from_trades_fangorn(threshold=1000):
+    """
+    Comptues the difference between the actual AUM and the one implied from trades
```
[the remark this answers is not in the document]
Comment (Arthurim) [reply src/fangorn/nav.py]: ✅

File: src/fangorn/nav.py [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+from sphinx.util import requests
+
+from src.fangorn.read import get_borrowing_terms_fangorn, get_pnl_spreadsheet_fangorn, get_cash_flows_fangorn
+from src.utils.constants import DATE_FORMAT
+
+
+def process_loan_to_cash_flows(start_date, counterparty, length_in_months, sym, amount, rate, loan_type):
+    """
+    Given some borrowing terms for a loan, compute the cashflows of the loan
+    Note that a loan can be BULLET or STANDARD (monthly repayements)
```
[the remark this answers is not in the document]
Comment (Arthurim) [reply src/fangorn/nav.py]: ✅

File: src/aeglos/read_write.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Arthurim) [reply src/aeglos/read_write.py]: ✅

File: src/fangorn/nav.py [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+    else:
+        raise Exception("Invalid loan type:", loan_type)
+
+
+def get_cash_flows_bullet_loan(start_date, counterparty, length_in_months, sym, amount, rate):
+    """
+    Given some borrowing terms, compute the cashflows of a bullet loan
+
+    return:pd.dataframe
+    columns = ['Date', 'Counterparty', 'Sym', 'Amount']
```
[the remark this answers is not in the document]
Comment (Arthurim) [reply src/fangorn/nav.py]: ✅

**Review by yvesleconte — DISMISSED** [review]

**Review by yvesleconte — APPROVED** [review]

**Pull request merged**